release-version.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/sh
```

Comment:
There's no need to commit `release-version.sh` to your own repository. The purpose of having it centrally in `scijava-scripts` is everybody can benefit from improvements and bugfixes by checking out the latest version from there. Local copies of this script will likely get out of sync with the SciJava-managed one.

Reply:
Thanks Jan, deleted the shell script locally, using the central script from the scijava repo instead.